odbc/src/api/data.rs

```diff
@@ -115,6 +115,22 @@ impl IntoCData for Bson {
             Bson::Int64(i) => Ok(i.to_le_bytes().to_vec()),
             Bson::Binary(b) => Ok(b.bytes),
             Bson::Decimal128(d) => Ok(d.bytes().to_vec()),
+            Bson::Document(d) => Ok(d.to_string().into_bytes()),
+            Bson::Array(a) => Ok(serde_json::to_string(
```

Comment:
Nice update to `Array`, it now shows as an array: `["a","b","c"]` instead of `abc`